contracts/GSN/bouncers/GSNBouncerERC20Fee.sol

```diff
@@ -27,11 +27,10 @@ contract GSNBouncerERC20Fee is GSNBouncerBase {
     __unstable__ERC20PrimaryAdmin private _token;
 
     /**
-     * @dev The arguments to the constructor are the details that the gas payment token will have: `name`, `symbol`, and
-     * `decimals`.
+     * @dev The arguments to the constructor are the details that the gas payment token will have: `name` and `symbol`.
      */
-    constructor(string memory name, string memory symbol, uint8 decimals) public {
-        _token = new __unstable__ERC20PrimaryAdmin(name, symbol, decimals);
+    constructor(string memory name, string memory symbol) public {
+        _token = new __unstable__ERC20PrimaryAdmin(name, symbol);
```

Comment:
fwiw I think this reads more clearly if the 18 is fixed here instead of in `__unstable__ERC20PrimaryAdmin`'s constructor.

Reply:
Hm, I suppose I agree. I was a bit thrown off by the `ERC20PrimaryAdmin` name and didn't put much thought into it. :sweat_smile: